<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StickRemoteControlDialog</class>
 <widget class="QDialog" name="StickRemoteControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Control</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="reflGroupBox">
       <property name="title">
        <string>Reflection Light</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QPushButton" name="reflSetPushButton">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="reflOffPushButton">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="reflCurrentLineEdit">
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="reflSpinBox">
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="reflCurrentLabel">
          <property name="text">
           <string>Current:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="reflOnPushButton">
          <property name="text">
           <string>On</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="sensorGroupBox">
       <property name="title">
        <string>Sensor Control</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="sensorStopPushButton">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sensorStartPushButton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="gainLabel">
            <property name="text">
             <string>Gain:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="gainComboBox">
            <property name="currentIndex">
             <number>8</number>
            </property>
            <item>
             <property name="text">
              <string>0.5x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>64x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>128x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>256x</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="intLabel">
            <property name="text">
             <string>Integration:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="intComboBox">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>100ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>300ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>400ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>600ms</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="agcCheckBox">
          <property name="text">
           <string>Automatic Gain Control</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reflReadPushButton">
          <property name="text">
           <string>Reflection Reading</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="readingGroupBox">
       <property name="title">
        <string>Sensor Readings</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="rawReadingLabel">
          <property name="text">
           <string>Raw:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="rawReadingLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="basicReadingLabel">
          <property name="text">
           <string>Basic:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="basicReadingLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>reflOffPushButton</tabstop>
  <tabstop>reflOnPushButton</tabstop>
  <tabstop>reflSpinBox</tabstop>
  <tabstop>reflSetPushButton</tabstop>
  <tabstop>sensorStopPushButton</tabstop>
  <tabstop>sensorStartPushButton</tabstop>
  <tabstop>gainComboBox</tabstop>
  <tabstop>intComboBox</tabstop>
  <tabstop>agcCheckBox</tabstop>
  <tabstop>reflReadPushButton</tabstop>
  <tabstop>rawReadingLineEdit</tabstop>
  <tabstop>basicReadingLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
